reference/catalog/products_catalog.v3.yml

```diff
@@ -1285,7 +1285,7 @@ paths:
         - Each image file or image uploaded by URL can be up to 8 MB.
       operationId: createProductImage
       parameters:
-        - $ref: '#/components/parameters/ContentType'
+        - $ref: '#/components/parameters/MultipartContentType'
```

Comment:
hey @bc-traciporter , just want to ensure you didnt miss that part ^^
```
 **Required Fields**
        - `image_file`, or
        - `image_url`
```
with `image_file` content type will be 'mutipart/form-data' but with `image_url` it will be  'application/json', so here we have a 2 payloads that depends on content type for single endpoint and it's fair to write about both of them.